### DISCUSSION TITLE
[Feature] does Immich support being hosted not on root?

### DISCUSSION THREAD

**Discussion by gardiol** [immich-app/immich#6329] [Feature Request]
### The feature

I self-host many services and protect them all behind a reverse proxy. I need to host immich under https://my-domain.xx/photos but i cannot find how to do this. Is it currently possible?

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
This isn't possible, you should use a subdomain instead. Hosting multiple different apps under folders like this also has security implications - they will be able to access eachothers cookies, local storage, etc.

**Comment by gardiol** [comment]
Not possible is incorrect: lots of apps support that (there are tons of examples, including big projects like PHP My Admin, RoundCube, DocuWiki...) so it can be done and it's not such a security issue when designed properly.

If it's not a priority i understand, but don't say not possible.
If it's not wanted, same, say so and not say not possible.

**Reply by bo0tzz** [reply]
Not possible [with immich right now], seeing as you asked whether this is currently possible [with immich right now] ;)

> it's not such a security issue when designed properly.

Can you elaborate on this?

Meanwhile, I'll close this as a duplicate of #1679
